test/e2e/backup_test.go

```diff
@@ -64,11 +64,11 @@ func verifyAWSEnvVars() error {
 // It returns the full S3 path where the backup is saved.
 func testEtcdBackupOperatorForS3Backup(t *testing.T) string {
 	f := framework.Global
-	suffix := fmt.Sprintf("-%d", rand.Uint64())
+	suffix := fmt.Sprintf("%d", rand.Uint64())
```

Comment:
can we not change the test code in this pr?
prefer just doc change.

Reply:
Yep. Let me separate them